config/templates/kava/evm/initstate/.kava/config/client.toml

```diff
@@ -0,0 +1,17 @@
+# This is a TOML config file.
+# For more information, see https://github.com/toml-lang/toml
+
+###############################################################################
+###                           Client Configuration                            ###
+###############################################################################
+
+# The network chain ID
+chain-id = ""
```

Comment:
does this need to be populated? This sets the default chain id for the cli right?